packages/core/config/babel-plugin-redwood-cell.js

```diff
@@ -0,0 +1,70 @@
+module.exports = function ({ types: t }) {
+  const names = [
+    'QUERY',
+    'Loading',
+    'Success',
+    'Failure',
+    'Empty',
+    'beforeQuery',
+    'afterQuery',
+  ]
+
+  let exportNames = []
+
+  return {
+    visitor: {
+      Program: {
+        exit(path) {
+          // import { withCell } from '@redwoodjs/web'
+          path.node.body.unshift(
+            t.importDeclaration(
+              [
+                t.importSpecifier(
+                  t.identifier('withCell'),
+                  t.identifier('withCell')
+                ),
+              ],
+              t.stringLiteral('@redwoodjs/web')
+            )
+          )
+
+          // export default withCell({ QUERY?, Loading?, Succes?, Failure?, Empty?, beforeQuery?, afterQuery? })
+          path.node.body.push(
+            t.exportDefaultDeclaration(
+              t.callExpression(t.identifier('withCell'), [
+                t.objectExpression(
+                  exportNames.map((name) =>
+                    t.objectProperty(
+                      t.identifier(name),
+                      t.identifier(name),
+                      false,
+                      true
+                    )
+                  )
+                ),
+              ])
+            )
+          )
+
+          exportNames = []
```

Comment:
This seems hacky, but I couldn't find another way to do this. Any suggestions?

Reply:
I think it's fine, maybe clarify why you're doing that with a comment.

One thing that might make this code read a bit easier is if you switched around `ExportNamedDeclaration` and `Program`

Reply:
Good idea! Will do.